src/components/map/components/legend/styles.scss

```diff
@@ -0,0 +1,26 @@
+div.uig-map-legend {
+  position: absolute;
+  z-index: 1;
+  display: flex;
+  background-color: #fff;
+  border: 1px #cbd2da solid;
```

Comment:
moeten we hier niet refereren naar kleuren en (padding/margin/...) groottes die bepaald zijn door vl-core?

Reply:
Naar analogie met de styles.scss van de map heb ik ook de $vl-border-color; gebruikt om de border van de map legend te kleuren, alle andere stijlen zijn gebleven omdat dit ook zo in de styles.scss van de map zo is. Indien de styling van de map centraler moet of de stijlen van vl-core moet gebruiken dan zal dit ook voor alle andere componenten van de map en de map zelf moeten gebeuren en dan zie ik dat meer als een apart ticket.